docs/input/docs/reference/build-servers/gitlab.md

```diff
@@ -7,3 +7,18 @@ Description: Details on the GitLab CI support in GitVersion
 To use GitVersion with GitLab CI, either use the [MSBuild
 Task](/docs/usage/msbuild) or put the GitVersion executable in your
 runner's `PATH`.
+
+A working example of integrating GitVersion with GitLab is maintained in the project [Utterly Automated Software and Artifact Versioning with GitVersion](https://gitlab.com/guided-explorations/devops-patterns/utterly-automated-versioning/)
```

Comment:
Can you please turn all links into reference links?

Reply:
I will need a little more context of what is meant by `reference links` - I couldn't find any information about it in CONTRIBUTING.md.

Reply:
Ahhh - you mean this: https://www.markdownguide.org/basic-syntax/#reference-style-links ?

Reply:
I have updated. I could find an example on the site. Pages like https://gitversion.net/docs/reference/modes/continuous-deployment seem to use inline links.

Reply:
Glad you figured it out. Some files still use inline links, I just want all new ones to use reference links. Every time I touch an `.md` file, I make sure to replace inline with reference. :)